programmers/난이도별/level01.하샤드_수/s2uyeoii.cpp

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+bool solution(int x) {
+
+    int sum = 0;
+    int x1 = x;
+
+    while(x>=1)
+    {
+       sum += x1%10;
+        x1/= 10;
+
+        if(x1 == 0)
+            break;
+
+        }
```

Comment:
```suggestion
    while(x>=1 || x1 != 0)
    {
       sum += x1 % 10;
        x1 /= 10;
    }
```
조건을 이렇게 줘도 되지 않을까요!?